CONTRIBUTING.md

```diff
@@ -73,9 +73,8 @@ scikit-bio is hosted on [GitHub](http://www.github.com), and we use GitHub's [Pu
 
 5. Install scikit-bio in "development mode" so that your changes are reflected in the installed package without having to reinstall the package each time:
```

Comment:
This text doesn't make a whole lot of sense now that we're referring users to another section. How about the following text?

```
5. Install scikit-bio for development. See [Setting up a development environment](#setting-up-a-development-environment).
```